composer.json

```diff
@@ -95,8 +93,14 @@
         }
     },
     "suggest": {
-        "doctrine/dbal": "Allow renaming columns and dropping SQLite columns.",
-        "guzzlehttp/guzzle": "Required for Mailgun and Mandrill mail drivers."
+        "aws/aws-sdk-php": "Required to use the SQS queue driver (~2.4).",
+        "doctrine/dbal": "Required to rename columns and drop SQLite columns (~2.4).",
+        "guzzlehttp/guzzle": "Required to use the Mailgun and Mandrill mail drivers (~5.0).",
+        "iron-io/iron_mq": "Required to use the iron queue driver (~1.5).",
+        "league/flysystem-aws-s3-v2": "Required to use the Flysystem S3 driver (~1.0).",
+        "league/flysystem-rackspace": "Required to use the Flysystem Rackspace driver (~1.0).",
+        "pda/pheanstalk": "Required to use the beanstalk queue driver (~3.0).",
+        "predis/predis": "Required to use the redis cache and queue drivers (~1.0)."
```

Comment:
@taylorotwell That's the point. I've moved it from require to suggest. I can revert that if you want. I just considered it intuitive otherwise people not using redis have to load quite a large library they don't need.